### ISSUE TITLE
[BUG] [Formatter]  hyperscript in script tags

### ISSUE THREAD

**Issue by johningve** [Riverside-Healthcare/djLint#697]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. ubuntu 22.04
 - Python Version (``python --version``) Python 3.11.3
 - djLint Version (``djlint --version``) djlint, version 1.31.0
 - template language: django

## Issue

Thanks for making this great tool!

I am using [hyperscript](https://hyperscript.org/) in my templates and running into problems.

Consider the following code:

```html
<div>
    <script type="text/hyperscript">
        on alert_success(value)
            render #alert_template with (class: 'alert-success', message: value)
            then put it into #alerts
        end
    </script>
</div>
```

When I format with djlint, the formatting of the hyperscript gets all messed up.
I don't think that djlint should touch these scripts at all.
To circumvent this, I am trying to get djlint to ignore the code like this:

```html
{# djlint: off #}
<div>
    <script type="text/hyperscript">
        on alert_success(value)
            render #alert_template with (class: 'alert-success', message: value)
            then put it into #alerts
        end
    </script>
</div>
{# djlint: on #}
```

When I do this, djlint no longer reformats the hyperscript, but for some reason, it unindents the closing `</script>` tag, like this:

```diff
@@ -5,6 +5,6 @@

             render #alert_template with (class: 'alert-success', message: value)
             then put it into #alerts
         end
-    </script>
+</script>
 </div>
 {# djlint: on #}
```

Here is my config:
```toml
[tool.djlint]
max_line_length = 100

ignore = "H031"

format_css = true
format_js = true
format_attribute_template_tags = false

preserve_blank_lines = true
```

## How To Reproduce

1. Copy the example code into a file.
2. Reformat with djlint.

**Comment by christopherpickering** [comment]
Hey @johningve the js formatter used here is [jsbeautify](https://github.com/beautify-web/js-beautify), I would suggest not using the `format-js` flag as they don't seem to support `hyperscript`.

![image](https://github.com/Riverside-Healthcare/djLint/assets/17788706/2b1b56a5-6469-4e46-b628-ce706e0d56cc)

I browsed around and can't find a formatter for `hyperscript`. If you know of one (python based preferable ;)) we can probably use it when the script type is set to `text/hyperscript`.

Sorry I don't have a better answer!

**Comment by johningve** [comment]
I guess my main concern was that djlint is changing the code that should be ignored. Like I mentioned in my example, the script tag that is within the ignored section is still being moved. I'm guessing that shouldn't happen?

**Comment by christopherpickering** [comment]
This should be fixed in the same update for #733.
